Teapot/Teapot.swift

```diff
@@ -53,10 +61,12 @@ open class Teapot {
     ///   - headerFields: A [String: String] dictionary mapping HTTP header field names to values. Defaults to nil.
     ///   - timeoutInterval: How many seconds before the request times out. Defaults to 15.0
     ///   - allowsCellular: a Bool indicating if this request should be allowed to run over cellular network or WLAN only.
+    ///   - deliveryQueue: a DispatchQueue where the completion block will be called. If none is supplied, uses the Teapot's defaultDeliveryQueue, which itself defaults to the main queue.
```

Comment:
FWIW you probably shouldn't mention what the default is for the `defaultDeliveryQueue` here - in theory that could change and then this would be incorrect. The caller should look at the docs for that rather than to each individual method

Reply:
👍